<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookManager</class>
 <widget class="QWidget" name="BookManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>25</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Manager">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>-1</y>
     <width>791</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>                                                                                                                                                                                       管理员：gkb</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>811</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QWidget" name="DataWidget" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>771</width>
     <height>541</height>
    </rect>
   </property>
   <widget class="QLabel" name="BookInformationManagerTitle">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>  图书信息管理</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>211</width>
     <height>601</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QLabel" name="Title">
      <property name="font">
       <font>
        <pointsize>22</pointsize>
        <italic>false</italic>
        <bold>true</bold>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>  图书管理系统</string>
      </property>
      <property name="textFormat">
       <enum>Qt::AutoText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BookInformationpushButton">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>图书信息管理</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BookClassifypushButton">
      <property name="font">
       <font>
        <pointsize>20</pointsize>
       </font>
      </property>
      <property name="text">
       <string>图书分类管理</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
